clang/include/clang/3C/ConstraintVariables.h

```diff
@@ -458,7 +468,7 @@ class FVComponentVariable {
   void mergeDeclaration(FVComponentVariable *From, ProgramInfo &I,
                         std::string &ReasonFailed);
   std::string mkItypeStr(const EnvironmentMap &E) const;
-  std::string mkTypeStr(const EnvironmentMap &E) const;
+  std::string mkTypeStr(const EnvironmentMap &E, bool EmitName) const;
```

Comment:
Is this common enough? should it get a default `= false`?

Reply:
Used once as `true` and once as `false`, so not much point in defaulting in either direction.